### DISCUSSION TITLE
Group view

### DISCUSSION THREAD

**Discussion by CFox243** [cmintey/wishlist#195] [Ideas]
View Type suggestion.
Ability to view all groups in one view.
To be able to group family members in groups and have a view to be able to see all groups.

[Family Fox] (group name)
------ Chris
------ Leanna
[Family Smith] (group name)
------ Member list 1
------ Member list 2
---------- Child Name Member list 2.1 (this could be a child of Group name depends how its implemented [Suggestion](https://github.com/cmintey/wishlist/discussions/194#discussion-7694464))
---------- Child Name Member list 2.2

**Comment by cmintey** [comment]
Hmm, interesting. I'll consider it, thanks!
